### ISSUE TITLE
Choose number not only by scrolling but also by tapping a visible number

### ISSUE THREAD

**Issue by Tiebo** [MarcinusX/NumberPicker#31]
Hey there, 

first of all: good job! 

I have a recommendation for improvement.
At the moment new values are selected by scrolling the numbers in the widget. It would be very helpful, if we could also tap on the next or previous visible number to select it.

For example, if I have 5 selected, then above is 4 and below the selected 5 is the 6. If I could just tap on the 4 to select it would help because then you wouldn't need to scroll just a tiny bit. 

Do you think this is possible? 

Thanks in advance, 

Tobi

**Comment by MarcinusX** [comment]
Hey,

I've implemented such feature in this [post](https://marcinszalek.pl/flutter/bmi-calculator-weight/) however, it introduced minor buggy moment when user clicks and during scrolling. I think it wouldn't be good to provide a feature and a bug at the same time so I didn't add it here.

If you have an idea how to do it properly based on what I did in that post, I would love to see it or (if you would like) I would love to accept a pull request :)

**Comment by MarcinusX** [comment]
Yeah, I had different behaviour then, so in that case it seems like you did it :)

**Comment by imaNNeoFighT** [comment]
@Tiebo If you did it, can you please make a pull request?

**Comment by dpedrinha** [comment]
Pull request please.

**Comment by MarcinusX** [comment]
Unfortunately, my way was buggy - at this moment there is no code for that.

**Comment by SilkeNL** [comment]
Was this merched? If so, how do I enable it?
